rts/motoko-rts/src/bitrel.rs

```diff
@@ -5,54 +5,60 @@ use crate::idl_trap_with;
 use crate::mem_utils::memzero;
 use crate::types::Words;
 
-const BITS: u32 = 2;
+const BITS: usize = 2;
 
 #[repr(packed)]
 pub struct BitRel {
     /// Pointer into the bit set
-    pub ptr: *mut u32,
+    pub ptr: *mut usize,
     /// Pointer to the end of the bit set
     /// must allow at least 2 * size1 * size2 bits
-    pub end: *mut u32,
-    pub size1: u32,
-    pub size2: u32,
+    pub end: *mut usize,
+    pub size1: usize,
+    pub size2: usize,
```

Comment:
These are just bounds on the size of the type tables, so probably could have remained u32, but maybe this is easier...

Reply:
I agree. I only changed this for symmetry/simplicity.

I will add it to the backlog list of changing it back to `u32`. If okay, we could do this at a later time.